setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = lto
-version = 0.0.16
+version = 0.0.17
```

Comment:
Do you update the version on every PR?

Reply:
Yap, but do you push after every PR?

Reply:
get to the point, why you're asking the same question? if i don't push when there are changes then the package will be outdated! i don't push for minor changes but in this case the lib won't work changing the module names

Reply:
1. It's not the same question - one is if you update version in the repo on every PR, the other one is if you push to PyPi on every PR.
2. Being outdated is fine, as long as it's versioned properly, that's why there are release notes (which don't correspond to 1 PR in the repo), that's the whole idea of having versions. No release notes and official docs in our case as it's still in beta though.
3. A simple answer of "No, I don't update if it's minor change, but for this one I do." would have been sufficient :).

Resolve the conversation when you read this comment if no questions on your side.

Reply:
@KaloyanTanev Continues integration with updating the version and pushing to PyPi is still on the todo. Please don't hold up a PR for this kind of discussion.